emukit/core/parameter_space.py

```diff
@@ -4,12 +4,18 @@
 import GPyOpt
 
 from . import ContinuousParameter
+from .discrete_parameter import DiscreteParameter, InformationSourceParameter
 
 
 class ParameterSpace(object):
     def __init__(self, parameters: List):
```

Comment:
return type annotation missing

Reply:
I think we can skip return type annotations for __init__ methods as they don't return anything

Reply:
OK, that is fine with me, but let's do it consistently then. 

I personally prefer uniform consistency over implications though, and adding a -> None once in a while does not add too much overhead and leaves the reader not as puzzled as of why something  was done/left out.

Reply:
Sure, let's agree on one and stick with it. I'll leave for a different issue/PR as I didn't touch this init method in this PR and there are quite a few that are inconsistent across the code. Feel free to open an issue!